app/src/main/java/org/walleth/ui/NetworkDefinitionViewHolder.kt

```diff
@@ -7,14 +7,22 @@ import org.walleth.data.networks.NetworkDefinition
 
 class NetworkDefinitionViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
-    fun bind(networkDefinition: NetworkDefinition, onClickAction: (entry: NetworkDefinition) -> Unit) {
+    fun bind(networkDefinition: NetworkDefinition, onClickAction: (entry: NetworkDefinition) -> Unit,
+             onInfoClickAction: (entry: NetworkDefinition, stats: Boolean) -> Unit) {
 
         itemView.network_title.text = networkDefinition.getNetworkName()
 
         itemView.setOnClickListener {
             onClickAction.invoke(networkDefinition)
         }
 
+        itemView.info_indicator.setOnClickListener {
+            onInfoClickAction.invoke(networkDefinition, false)
+        }
+
+        itemView.stats_indicator.setOnClickListener {
+            onInfoClickAction.invoke(networkDefinition, true)
```

Comment:
I think we should have 2 lambdas here for readability - onInfoClick and onStatsClick